exampleSite/config.toml

```diff
@@ -73,3 +73,8 @@ weight = 20
 name = "<i class='fas fa-fw fa-bullhorn'></i> Crédits"
 url = "/credits"
 weight = 30
+
+[Languages.zh]
+title = "Hugo 主题的 Learn 文档"
```

Comment:
Could you add shortcuts like for the other languages ?

Reply:
Cool. I'll do it.